packages/chat-widget/src/index.tsx

```diff
@@ -554,6 +554,7 @@ export const Widget = forwardRef<WidgetRef, Props>(function Widget(props, ref) {
             <C.PinBubble
               isActive={!expanded && isNudgeVisible}
               onClick={() => {
+                props.onNudgeClose?.(chat.conversationHandler);
```

Comment:
Should this not also be called when the timeout triggers?

Reply:
I don't think so, they want to know how many times the user dismisses it explicitly, for analytics purposes. We might need separate events but I would rather kick that can down the road.

Reply:
Maybe that should be then documented. From the current docs, I wouldn't understand that it's just for manual closes.

Reply:
Pushes a commit for the docs.